src/topojson/topojson.ml

```diff
@@ -380,10 +380,10 @@ module Make (J : Intf.Json) = struct
       objects : (string * Geometry.t) list;
       arcs : Geometry.Position.t array array;
       foreign_members : (string * json) list;
-      transform : string * Geometry.Position.t array;
+      transform : transform option;
     }
 
-    type transform = { scale : float * float; translate : float * float }
+    and transform = { scale : float array; translate : float array }
```

Comment:
This should be `float * float` and not an array -- JSON doesn't have a way of expressing pairs and that's why the TopoJSON spec uses an array, but we can convert to a pair.